### PR TITLE
Feature/qu 927 beta into develop

### PR DESCRIPTION
**Pull request by jamesjacobs**
Merging Pulsar -beta branch.

### CODE REVIEW

File: images/branding/logo-cxm-white.svg [hunk 1]
```diff
@@ -1,4 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
+<<<<<<< HEAD
```
Comment (langy) [inline images/branding/logo-cxm-white.svg]: Merge conflict?

File: bower.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
     "select2": "~4.0.0",
     "sticky": "latest",
     "store.js": "latest",
+    "susy" : "latest",
```
Comment (Stanton) [inline bower.json]: I know other's have 'latest', but new deps should be locked to a version (using `--save-dev`)

File: js/main.js [hunk 3]
```diff
@@ -19,6 +22,8 @@
 
         pulsar.button.init();
         pulsar.flash.init();
+        pulsar.helpText.init();
+        pulsar.helpText.updateHelpSidebar();
```
Comment (Stanton) [inline js/main.js]: you just *had* to be different, huh?

File: js/require-pulsar.js [hunk 4]
```diff
@@ -233,7 +233,7 @@ define([
             // actionsMenu();
 
             // Refresh positions of sticky elements
-            console.log($('.is-sticky'));
+            //console.log($('.is-sticky'));
```
Comment (Stanton) [inline js/require-pulsar.js]: remove

File: stylesheets/_component.tab-help.scss [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        &:hover {
+            text-decoration: none;
+        }
+    }
+}
+
+.show-page-help {
+    -webkit-transition: all 100ms ease-in;
+    border-radius: 4px;
+    color: $jadu-copy-gray;
```
Comment (Stanton) [inline stylesheets/_component.tab-help.scss]: should use colour from base palette

File: stylesheets/_component.tab-help.scss [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    text-decoration: none;
+    transition: all 100ms ease-in;
+
+    @include respond-min($screen-desktop) {
+        display: none;
+    }
+
+    &:hover,
+    &.is-open {
+        color: #2d89b7;
```
Comment (Stanton) [inline stylesheets/_component.tab-help.scss]: use `$nav-color-dark`?

File: stylesheets/_component.tab-help.scss [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    &:active {
+        color: darken(#2d89b7, 10%);
```
Comment (Stanton) [inline stylesheets/_component.tab-help.scss]: use `$nav-color-dark`?

File: stylesheets/_component.forms.scss [hunk 8]
```diff
@@ -53,6 +47,75 @@ fieldset {
...
+            border-radius: 0;
+        }
+    }
+
+    // IE8 - position text in the middle of inputs
+    @at-root {
+        input[type="text"]#{&},
+        input[type="url"]#{&},
+        input[type="password"]#{&},
+        input[type="email"]#{&} {
```
Comment (langy) [inline stylesheets/_component.forms.scss]: http://html5doctor.com/html5-forms-input-types/ - no biggie right now but there's a few types missing that might bring bugs later

File: stylesheets/_component.tab-help.scss [hunk 9]
```diff
@@ -0,0 +1,101 @@
+$nav-color-light: #3399cc !default;
+$nav-color-dark:  #2d89b7 !default;
+
+.tab-help-container {
+
+    -webkit-backface-visibility: hidden;
+    -webkit-transform: translate3d(-100%, 0, 0);
+    backface-visibility: hidden;
```
Comment (langy) [inline stylesheets/_component.tab-help.scss]: Is this needed as well as the -webkit- one above?

File: stylesheets/_component.tab-help.scss [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+    background: $nav-color-dark;
+    box-shadow: inset 4px 0 0 darken($nav-color-dark, 5%);
+    color: $white;
+    display: table-cell;
+    height: 100%;
+    right: -600px;
+    position: absolute;
+    top: 0;
+    transform: translate3d(-100%, 0, 0);
```
Comment (langy) [inline stylesheets/_component.tab-help.scss]: Is this needed as well as the -webkit- one above?

File: stylesheets/_component.tab-help.scss [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+
+    .lt-ie10 & {
+        right: -300px;
+    }
+
+    .open-help & {
+        -webkit-transform: translate3d(0, 0, 0);
+        filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=100);
+        opacity: 1;
+        transform: translate3d(0, 0, 0);
```
Comment (langy) [inline stylesheets/_component.tab-help.scss]: Is this needed as well as the -webkit- one above?

File: stylesheets/_component.tab-help.scss (hunk 9 above)
Comment (Stanton) [inline stylesheets/_component.tab-help.scss]: Pulsar uses autoprefixer, so `-webkit` rules can likely be removed

File: stylesheets/_component.tab-help.scss [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+    }
+}
+
+// hide help button on mobile when no JS
+
+.tab-help  {
+    -webkit-transform-origin: 50% 0%;
+    -webkit-transform: translate3d(70%, 0, 0) scale3d(0.9, 0.9, 0.9);
+    -webkit-transition-delay: ease, 0s;
+    -webkit-transition: opacity 300ms 100ms, -webkit-transform 400ms ease;
```
Comment (langy) [inline stylesheets/_component.tab-help.scss]: Could we not let autoprefixer take care of these via the 'proper' versions below?

File: stylesheets/_deprecated.styles.scss [hunk 13]
```diff
@@ -392,3 +394,357 @@ $toolbar-notification-item-hover: '#f00';
...
+        border-color: transparent;
+        &:hover {
+          background-color: $gray-lighter;
+        }
+      }
+    }
+
+    .tabs__list[data-mobile-togglable] > li > .tabs__list {
+      background-color: $accent-color;
+      border: 0 none;
```
Comment (langy) [inline stylesheets/_deprecated.styles.scss]: Just 0 will do

File: stylesheets/_deprecated.styles.scss [hunk 14]
```diff
@@ -392,3 +394,357 @@ $toolbar-notification-item-hover: '#f00';
...
+      padding: 0;
+
+      > li > a {
+        border: 0 none;
```
Comment (langy) [inline stylesheets/_deprecated.styles.scss]: And here

File: stylesheets/_structure.grid-mixins.scss [hunk 15]
```diff
@@ -1,85 +0,0 @@
...
-            @include inner-grid($width, 5, 6);
-        }
-        .six-sevenths {
-            @include inner-grid($width, 6, 7);
-        }
-        .seven-eigths {
-            @include inner-grid($width, 7, 8);
-        }
-    }
-}
```
Comment (langy) [inline stylesheets/_structure.grid-mixins.scss]: https://i.imgflip.com/zt22h.jpg

File: stylesheets/_structure.grid.scss [hunk 16]
```diff
@@ -1,158 +1,35 @@
...
-        height: 200px;
-        background: rgb(180,200,240);
-    }
-}
+// Susy grid settings
+$susy: (
+    columns: 12,
+    container: 100%,
+    gutters: .25,
+);
```
Comment (langy) [inline stylesheets/_structure.grid.scss]: <3

File: stylesheets/_component.forms.scss (hunk 8 above)
Comment (Stanton) [inline stylesheets/_component.forms.scss]: Do a separate ticket for that one

File: stylesheets/_utility.misc.scss [hunk 17]
```diff
@@ -95,3 +95,11 @@
     vertical-align: top !important;
 }
 
+.visibility-hidden {
+    visibility: hidden;
+
+    .lt-ie10 & {
+        display: none;
```
Comment (langy) [inline stylesheets/_utility.misc.scss]: opacity: 0; is probably a nicer alternative for this

Comment (Stanton) [inline stylesheets/_utility.misc.scss]: Will still need to work for IE8 tho

Comment (langy) [inline stylesheets/_utility.misc.scss]: Tru dat, ignore

File: stylesheets/_component.tab-help.scss (hunk 10 above)
Comment (jamesjacobs) [inline stylesheets/_component.tab-help.scss]: Removed webkit, trusting in autoprefixer.

**Pull request merged**